### ISSUE TITLE
Problems with testing

### ISSUE THREAD

**Issue by kmdrGroch** [stryker-mutator/stryker#1586]
Hello again dear Nico and Simondel,
Since again I have some time to help with your project I started with cloning installing and running your code. And there is a problem,

It crashed 3 times in a row (first time probably coz I haven't used 'npm run build')
the second one was of course with timeout in stryker\stryker\packages\webpack-transpiler\test\unit\compiler\WebpackCompiler.spec.js,
but the last one is super strange:
```
ChildProcessProxy
@stryker-mutator/core:        should throw an OutOfMemoryError if the process went out-of-memory:
@stryker-mutator/core:       AssertionError: expected promise to be rejected with 'OutOfMemoryError' but it was
rejected with 'Error: Child process [pid 5500] exited unexpectedly with exit code 134 (without signal). Last part of stdout and stderr was:\r\n\t\r\n\tWriting Node.js report to file: report.20190616.115549.5500.0.001.json\r\n\t\r\n\tNode.js report completed\r\n\t\r\n\t 1: 00007FF67F31D7AA public: __cdecl v8::internal::GCIdle\r\n\tTimeHandler::GCIdleTimeHandler(void) __ptr64+4618\r\n\t 2: 00007FF67F2CB736 uv_loop_fork+86646\r\n\t 3: 00007FF67F2CC1FD uv_loop_fork+89405\r\n\t\r\n\t 4: 00007FF67F6F454E void __cdecl v8::internal::FatalP\r\n\trocessOutOfMemory(class v8::internal::Isolate * __ptr64,char const \r\n\t* __ptr64)+798\r\n\t 5: 00007FF67F6F4487 void __cde\r\n\tcl v8::internal::FatalProcessOutOfMemory(class v8::internal::Isolate \r\n\t* __ptr64,char const * __ptr64)+599\r\n\t\r\n\t 6: 00007FF67F7A7F64 public: static bool __cdecl v\r\n\t8::internal::Heap::RootIsImmortalImmovable(int)+14068\r\n\t\r\n\t 7: 00007FF67F7A5FFB public: static bool \r\n\t__cdecl v8::internal::Heap::RootIsImmortalIm\r\n\tmovable(int)+6027\r\n\t 8: 00007\r\n\tFF67F8DDC6D private: class v8::internal::HeapObject * __ptr64 __cdecl v8::\r\n\tinternal::Factory::AllocateRawArray(int,enum v8::internal::PretenureFlag) __ptr64+61\r\n\t\r\n\t 9: 00007FF67F8DE602 private: class v8::interna\r\n\tl::Handle<class v8::internal::FixedArray> __cdec\r\n\tl v8::internal::Factory::NewFixedArrayWithFiller(enum v8::intern\r\n\tal::Heap::RootListIndex,int,class v8::internal::Objec\r\n\tt * __ptr64,enum v8::internal::PretenureFlag) __ptr64+66\r\n\t\r\n\t10: 00007FF67F9218DA public: static class v8::i\r\n\tnternal::wasm::AsmType * __ptr64 __cdecl v8::internal::wasm\r\n\t::AsmType::Void(void)+71658\r\n\t11: 00007FF67F\r\n\t90F163 class v8::internal::MaybeHandle<class v8::i\r\n\tnternal::BigInt> __cdecl v8::internal::BigIntLite\r\n\tral(class v8::internal::Isolate * __ptr64,char const * _\r\n\t_ptr64)+133555\r\n\t12: 00007FF67F9A0DA0 public: static i\r\n\tnt __cdecl v8::internal::StoreBuffer::StoreBuffer\r\n\tOverflow(class v8::internal::Isolate * __ptr64)+64256\r\n\t1\r\n\t3: 00000399B2550481 \r\n\t'
@stryker-mutator/core:       + expected - actual
@stryker-mutator/core:       -Error: Child process [pid 5500] exited unexpectedly with exit code 134 (without signal). Last part of stdout and stderr was:
@stryker-mutator/core:       -
@stryker-mutator/core:       -  Writing Node.js report to file: report.20190616.115549.5500.0.001.json
@stryker-mutator/core:       -
@stryker-mutator/core:       -  Node.js report completed
@stryker-mutator/core:       -
@stryker-mutator/core:       -   1: 00007FF67F31D7AA public: __cdecl v8::internal::GCIdle
@stryker-mutator/core:       -  TimeHandler::GCIdleTimeHandler(void) __ptr64+4618
@stryker-mutator/core:       -   2: 00007FF67F2CB736 uv_loop_fork+86646
@stryker-mutator/core:       -   3: 00007FF67F2CC1FD uv_loop_fork+89405
@stryker-mutator/core:       -
@stryker-mutator/core:       -   4: 00007FF67F6F454E void __cdecl v8::internal::FatalP
@stryker-mutator/core:       -  rocessOutOfMemory(class v8::internal::Isolate * __ptr64,char const
@stryker-mutator/core:       -  * __ptr64)+798
@stryker-mutator/core:       -   5: 00007FF67F6F4487 void __cde
@stryker-mutator/core:       -  cl v8::internal::FatalProcessOutOfMemory(class v8::internal::Isolate
@stryker-mutator/core:       -  * __ptr64,char const * __ptr64)+599
@stryker-mutator/core:       -
@stryker-mutator/core:       -   6: 00007FF67F7A7F64 public: static bool __cdecl v
@stryker-mutator/core:       -  8::internal::Heap::RootIsImmortalImmovable(int)+14068
@stryker-mutator/core:       -
@stryker-mutator/core:       -   7: 00007FF67F7A5FFB public: static bool
@stryker-mutator/core:       -  __cdecl v8::internal::Heap::RootIsImmortalIm
@stryker-mutator/core:       -  movable(int)+6027
@stryker-mutator/core:       -   8: 00007
@stryker-mutator/core:       -  FF67F8DDC6D private: class v8::internal::HeapObject * __ptr64 __cdecl v8::
@stryker-mutator/core:       -  internal::Factory::AllocateRawArray(int,enum v8::internal::PretenureFlag) __ptr64+61
@stryker-mutator/core:       -
@stryker-mutator/core:       -   9: 00007FF67F8DE602 private: class v8::interna
@stryker-mutator/core:       -  l::Handle<class v8::internal::FixedArray> __cdec
@stryker-mutator/core:       -  l v8::internal::Factory::NewFixedArrayWithFiller(enum v8::intern
@stryker-mutator/core:       -  al::Heap::RootListIndex,int,class v8::internal::Objec
@stryker-mutator/core:       -  t * __ptr64,enum v8::internal::PretenureFlag) __ptr64+66
@stryker-mutator/core:       -
@stryker-mutator/core:       -  10: 00007FF67F9218DA public: static class v8::i
@stryker-mutator/core:       -  nternal::wasm::AsmType * __ptr64 __cdecl v8::internal::wasm
@stryker-mutator/core:       -  ::AsmType::Void(void)+71658
@stryker-mutator/core:       -  11: 00007FF67F
@stryker-mutator/core:       -  90F163 class v8::internal::MaybeHandle<class v8::i
@stryker-mutator/core:       -  nternal::BigInt> __cdecl v8::internal::BigIntLite
@stryker-mutator/core:       -  ral(class v8::internal::Isolate * __ptr64,char const * _
@stryker-mutator/core:       -  _ptr64)+133555
@stryker-mutator/core:       -  12: 00007FF67F9A0DA0 public: static i
@stryker-mutator/core:       -  nt __cdecl v8::internal::StoreBuffer::StoreBuffer
@stryker-mutator/core:       -  Overflow(class v8::internal::Isolate * __ptr64)+64256
@stryker-mutator/core:       -  1
@stryker-mutator/core:       -  3: 00000399B2550481
@stryker-mutator/core:       -
@stryker-mutator/core:       +OutOfMemoryError
```

**Comment by nicojs** [comment]
I think this has to do with wct only supporting java 8 and below. You're probably on a newer version. I kind of have a love-hate relationship with WCT, [as you can see with the amount of issues reported by me](https://github.com/Polymer/tools/issues/created_by/nicojs). 

Apparently, [wctlocal@2.1.4 should support Java 8+](https://github.com/Polymer/tools/issues/3398#issuecomment-499601025).

We're happy to accept PR's for this issue 👍

**Comment by nicojs** [comment]
Wouw!! I had this issue yesterday as wel. It was a problem with Stryker! It occurred on Windows with Node12. 

I fixed it in #1635. If not, please open a new issue. Thanks!
